mono/metadata/image.c

```diff
@@ -520,7 +520,7 @@ load_metadata_ptrs (MonoImage *image, MonoCLIImageInfo *iinfo)
 			image->heap_tables.size = read32 (ptr + 4);
 			ptr += 8 + 3;
 			image->uncompressed_metadata = TRUE;
-			mono_trace (G_LOG_LEVEL_INFO, MONO_TRACE_ASSEMBLY, "Assembly '%s' has the non-standard metadata heap #-.\nRecompile it correctly (without the /incremental switch or in Release mode).", image->name);
+			mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_ASSEMBLY, "Assembly '%s' has the non-standard metadata heap #-.\nRecompile it correctly (without the /incremental switch or in Release mode).", image->name);
```

Comment:
INFO?

Reply:
This one might be ok as `DEBUG`. These `#-` things will show up once we're able to consume EnC deltas, too.  And it doesn't look like it's harmful.

Reply:
Hrm, okay. I was also unsure on the problematic image stuff lower in this file, but I assume that all will be logged later on anyway if the load fails?